data_structures/src/chain.rs

```diff
@@ -1471,7 +1474,7 @@ impl TransactionsPool {
     ///
     /// assert_eq!(pool.vt_len(), 0);
     ///
-    /// pool.insert(transaction);
+    /// pool.insert(transaction,0);
```

Comment:
May I ask for 1 extra space in all the doc examples?

```suggestion
    /// pool.insert(transaction, 0);
```